lib/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@storyblok/nuxt",
+  "version": "2.0.0",
+  "description": "Storyblok Nuxt.js module",
+  "license": "MIT",
+  "contributors": [
+    {
+      "name": "Alexander Feiglstorfer <delooks@gmail.com>"
+    }
+  ],
+  "main": "./module.js",
+  "repository": "https://github.com/storyblok/storyblok-nuxt",
+  "scripts": {
+    "test": "npm run test:unit && npm run test:e2e",
+    "test:unit": "jest __tests__",
+    "cy:open": "cypress open",
+    "cy:run": "cypress run",
+    "playground:run": "cd ../playground && npm run start",
+    "pretest:e2e": "cd ../playground && npm run build",
+    "test:e2e": "start-server-and-test playground:run http://localhost:3000 cy:run",
+    "test:e2e-watch": "start-server-and-test playground:run http://localhost:3000 cy:open",
+    "prepublishOnly": "npm run build && cp ../README.md ./"
```

Comment:
I believe this `prepublishOnly` script will failed because we don't have a `build` script.

Reply:
You're right! In v2 we don't need the build so we can remove it